packages/contentful--create-contentful-app/package.json

```diff
@@ -36,10 +37,8 @@
     "@contentful/cra-template-create-contentful-app": "file:../contentful--cra-template-create-contentful-app",
```

Comment:
we might need to remove this

Reply:
Yes forgot about that and did it in the follow-up PR: https://github.com/contentful/create-contentful-app/pull/846/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L52